### PR TITLE
Bug 1154195 - [System] split the visual from browser_context_menu.js

### PR DESCRIPTION
**Pull request by huchengtw-moz**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a23fee2f911785f3621ed45eb2a937eb69772915

File: apps/system/js/app_chrome.js [hunk 1]
```diff
@@ -321,7 +321,7 @@
   };
 
   AppChrome.prototype.titleClicked = function ac_titleClicked() {
-    if ((Service && Service.locked) || this.app.contextmenu.isVisible()) {
+    if ((Service && Service.locked) || this.app.contextmenu.isShown()) {
```
Comment (huchengtw-moz) [inline apps/system/js/app_chrome.js]: The `isVisible` is the same as `isShown`. I had merged it into isShown.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a8e6684dc2bae9ee8e4a76725a9437e64570e081

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5e5197d44a1dc4250f3dfc05c705199320a973c1

**Pull request merged**